tidb-cloud/migrate-from-amazon-s3-or-gcs.md

```diff
@@ -61,21 +61,27 @@ For detailed steps, see [Configure Amazon S3 access](/tidb-cloud/config-s3-and-g
 
 ### Step 3. Import data into TiDB Cloud
 
-1. On the **Data Import Task** page, besides the **Role ARN** field, you also need to fill in the following information:
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/), and navigate to the **Clusters** page.
+
+2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
+
+3. On the **Data Import** page, fill in the following information:
 
-    - **Data Source Type**: `AWS S3`.
-    - **Bucket URL**: fill in the bucket URL of your source data.
     - **Data Format**: choose the format of your data.
-    - **Target Cluster**: fill in the **Username** and **Password** fields.
-    - **DB/Tables Filter**: if necessary, you can specify a [table filter](/table-filter.md#syntax). If you want to configure multiple filter rules, use `,` to separate the rules.
+    - **Location**: `AWS`
+    - **Bucket URL**: fill in the bucket URL of your source data.
+    - **Role-ARN**: enter the Role-ARN you obtained in [Step 2](#step-2-configure-amazon-s3-access).
+    - **Target Cluster**: shows the cluster name and the region name.
 
-2. Click **Import**.
+    If the region of the bucket is different from your cluster, confirm the compliance of cross region. Click **Next**.
+    
+    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```

Comment:
```suggestion
    TiDB Cloud starts validating whether it can access your data in the specified bucket URL. After validation, TiDB Cloud tries to scan all the files in the data source using the default file naming pattern, and returns a scan summary result on the left side of the next page. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
```